external/twitter/src/main/scala/org/apache/spark/streaming/twitter/TwitterUtils.scala

```diff
@@ -40,7 +40,7 @@ object TwitterUtils {
       twitterAuth: Option[Authorization],
       filters: Seq[String] = Nil,
       storageLevel: StorageLevel = StorageLevel.MEMORY_AND_DISK_SER_2
-    ): ReceiverInputDStream[Status] = {
+    ): ReceiverInputDStream[Status] = ssc.withScope {
```

Comment:
Alright, I implemented what you suggested in the latest commit. I had to add another helper method that `InputDStream` overrides to include the ID. For most streams there is not much duplication, but for `FileInputDStream`, for instance, we have to put `"file stream"` in 3 places. I think there are dis/advantages in both approaches. Let me know what you think about the latest one.